src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => {};
+import { User } from '../../types';
+
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
```

Comment:
Even if user is null, you will render empty `a` tag - why not just return `null`?